Gemfile

```diff
@@ -5,5 +5,5 @@ gem 'cocoapods', '0.36.1'
 gem 'cocoapods-keys', :git => 'https://github.com/orta/cocoapods-keys.git'
 
 gem 'xcpretty'
-gem 'shenzhen'
 gem 'sbconstants'
+gem 'fastlane', :git => 'https://github.com/ashfurrow/fastlane.git'
```

Comment:
Will update once [this](https://github.com/KrauseFx/fastlane/pull/168) is resolved.